build/karma.conf.babel.js

```diff
@@ -42,23 +42,19 @@ module.exports = (karmaConfig) => {
     },
     webpack: {
       devtool: 'cheap-module-source-map',
-      module: {
-        ...webpackConfig.module,
+      module: Object.assign({}, webpackConfig.module, {
```

Comment:
OK, it works with arrays.  Also seems to be with rest params, but **not** with objects....

![image](https://cloud.githubusercontent.com/assets/5067638/16511171/6475f390-3f02-11e6-8581-4de7f818f47d.png)

source: http://node.green/